cpp/validate_harvested_records.cc

```diff
@@ -76,35 +77,85 @@ struct FieldInfo {
 };
 
 
-class JournalInfo {
-    std::string zeder_id_;
-    std::string zeder_instance_;
-    bool not_in_database_yet_;
+struct GeneralInfo {
```

Comment:
I think this can be a class, also maybe we can find a more specific name?

Reply:
Added a comment, not sure what a better name would be